distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+.*NIX
+----
+unzip ${branding-lowercase}-X.Y.zip
+----
+
+.Example: Use Java to Unzip in Windows
+----
+"C:\Program Files\Java\jdk1.8.0\bin\jar.exe" xf ${branding-lowercase}-X.Y.zip
+----
+
+* Run ${branding} using the appropriate script.
+
+.*NIX
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-X.Y/bin/${branding-lowercase}
+----
+
+.Windows
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-X.Y/bin/${branding-lowercase}.bat
+----
+
+Execute the following command at the command line for status:
+
+.View Status
+----
+${branding-lowercase}${at-symbol}local>list
+----
+
+// Verify this step is still required.
+* If the ${branding} Standalone Solr Server will be installed later, an additional configuration step is required for the ${branding} kernel.
+Add the following lines to the bottom of the `<INSTALL_DIR>/etc/org.ops4j.pax.web.cfg` file:
+
+.Additional Configuration Step
+----
+# Jetty Configuration
+`org.ops4j.pax.web.config.file=<KARAF.HOME>/etc/jetty.xml`
+----
+
+* Run the ${branding} using the appropriate script.
+
+.*NIX
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}
+----
+
+.Windows
+----
+<INSTALL_DIRECTORY>/${branding-lowercase}-${project.version}/bin/${branding-lowercase}.bat
+----
+
+The distribution takes a few moments to load depending on the hardware configuration.
+Execute the following command at the command line for status:
+
+.View Status
+----
+${branding}${at-symbol}local>list
+----
+The list of bundles should look similar to this:
+
+.${branding} List of Apps Installed
+----
+.${branding}${at-symbol}local>list
+START LEVEL 100 , List Threshold: 50
+   ID   State         Blueprint      Spring    Level  Name
+[ 111] [Active     ] [            ] [       ] [   80] Commons IO (2.1.0)
+[ 113] [Active     ] [Created     ] [       ] [   80] ${branding} :: Distribution :: Console Branding Plugin (${project.version})
+----
+
+.${branding} Application Installation Dependencies
+
+[WARNING]
+====
+If completing a non-standard installation, be aware that some applications depend on other ${branding} applications being installed.
+
+This hierarchy can be shown using the `app:tree` command
+
+[source]
+----
+${ddf-branding-lowercase}${at-symbol}local>app:tree
++- opendj-embedded
++- platform-app
+|   +- catalog-app
+|   |   +- search-ui-app
+|   |   |   +- spatial-app
+|   |   +- solr-app
+|   +- security-services-app
+|   |   +- admin-app
+----
+
+====
+
+==== Verifying Installation
+
+At this point, ${branding} should be configured and running with a Solr Catalog Provider.
+New features (endpoints, services, and sites) can be added as needed.
+
+Verification is achieved by checking that all of the ${branding} bundles are in an Active state (excluding fragment bundles which remain in a Resolved state).
+
+Execute the following command to display the status of all the ${branding} bundles:
```

Comment:
We may want to add a note here saying that DDF can take a little while to fully start and that it may be necessary to wait a few minutes before running the `list` command.